src/settings/components/TermsConfig/TermsConfig.css

```diff
@@ -0,0 +1,5 @@
+@import "@folio/stripes-components/lib/variables";
```

Comment:
Can you instead put this in [a generic styles.css and import that file? We've started doing that in ui-agreements](https://github.com/folio-org/ui-erm/blob/1c57c57237a946e9bf756e9c4d809a87b25977bd/src/components/LicensesFieldArray/LicenseField.js#L14) and stripes-erm-components because otherwise we end up getting a bunch of these chintzy CSS files scattered around.

Reply:
I'm waiting for Gill's decision before I fix this.
If we dump the red bold **Error** we don't need a styles.css at all

Reply:
I added styles.css to `src/settings/components` instead of `src/components` (like in ui-agreements) because I couldn't import it from there. I understood that this has something to do with the rules of import extensions, but didn't know how to change that.

Reply:
I added styles.css to `src/settings/components` instead of `src/components` (like in ui-agreements) because I couldn't import it from there. I understood that this has something to do with the rules of import extensions, but didn't know how to change that.